runtime/lua/vim/_meta/vvars.lua

```diff
@@ -199,7 +238,7 @@ vim.v.errors = ...
 ---   reason           Reason for completion being done. `CompleteDone`
 ---   complete_word    The word that was selected, empty if abandoned complete.
 ---   complete_type    See `complete_info_mode`
---- @type any
+--- @type vim.v.event
```

Comment:
Instead of using `@type`, you can inline for the full class definition here. That will then make the above text the description of the class.